apps/website/docs/react/api/merge-classes.md

```diff
@@ -112,6 +112,25 @@ function App(props) {
 }
 ```
 
+### Combining Griffel classes with custom class strings
+
+You may find yourself in a position where your component accepts classes via props or have global classes that you need applied to the same elements being styled by Griffel. For this, you can also use `mergeStyles`: 
+
+```jsx
+const useClasses = makeStyles({
+  root: { color: 'red' }
+});
+
+function App(props) {
+  const classes = useClasses();
+  const rootClass = mergeClasses(classes.root, "custom-global-class1 custom-global-class2");
+
+  return (
+     <div className={rootClass}>...</div>
+  )
+}
+```
```

Comment:
I just realized that we have already an example for this case above "Incorrect usages":

<img width="800" alt="image" src="https://github.com/microsoft/griffel/assets/14183168/202aef7b-1895-4bdc-88a4-0f8dfd0ea6d6">

I suggest to modify that section and add above it:

```
### Merging plain CSS classes

Your component may accept classes via props or have static classes that you need applied to the same elements being styled by Griffel. For this, you can also use `mergeClasses`:
```

WDYT?

Reply:
Sounds good!

Reply:
@benkeen would you be able to do the change? 🐱

Reply:
Yup absolutely! Sorry - last night was busy. Will update it tonight.

Reply:
Updated. Definitely better, thanks!